### ISSUE TITLE
Add parser for FSCache

### ISSUE THREAD

**Issue by francescsanjuanmrf** [prometheus/procfs#259]
If fscache is enabled (https://www.kernel.org/doc/Documentation/filesystems/caching/fscache.txt) stats will be found in /proc/fs/fscache/stats

Will be great to have this data available.

This is an example of the output:
```
FS-Cache statistics
Cookies: idx=3 dat=67877 spc=0
Objects: alc=67473 nal=0 avl=67473 ded=388
ChkAux : non=0 ok=0 upd=0 obs=0
Pages  : mrk=547164 unc=364577
Acquire: n=67880 nul=0 noc=0 ok=67880 nbf=0 oom=0
Lookups: n=67473 neg=67473 pos=0 crt=67473 tmo=0
Invals : n=14 run=14
Updates: n=0 nul=0 run=14
Relinqs: n=394 nul=0 wcr=0 rtr=0
AttrChg: n=0 ok=0 nbf=0 oom=0 run=0
Allocs : n=0 ok=0 wt=0 nbf=0 int=0
Allocs : ops=0 owt=0 abt=0
Retrvls: n=151959 ok=82823 wt=23467 nod=69136 nbf=0 int=0 oom=0
Retrvls: ops=151959 owt=42747 abt=0
Stores : n=225565 ok=225565 agn=0 nbf=0 oom=0
Stores : ops=69156 run=294721 pgs=225565 rxd=225565 olm=0
VmScan : nos=364512 gon=0 bsy=0 can=0 wt=0
Ops    : pend=42753 run=221129 enq=628798 can=0 rej=0
Ops    : ini=377538 dfr=27 rel=377538 gc=27
CacheOp: alo=0 luo=0 luc=0 gro=0
CacheOp: inv=0 upo=0 dro=0 pto=0 atc=0 syn=0
CacheOp: rap=0 ras=0 alp=0 als=0 wrp=0 ucp=0 dsp=0
CacheEv: nsp=0 stl=0 rtr=0 cul=0

**Comment by francescsanjuanmrf** [comment]
I write a PR (https://github.com/prometheus/procfs/pull/262) for this enhancement. But codespell tests are failing. The problematic words are keywords to parse /proc/fs/fscache/Stats output.
Can anyone review it and check if it is ok? I'm open to any change.

**Comment by pgier** [comment]
Fixed by #262
